railcontrol.conf.dist

```diff
@@ -8,6 +8,10 @@ dbfilename = railcontrol.sqlite
 # Default dbkeepbackups is 10
 dbkeepbackups = 10
 
+# If this is set to :: the webserver will listen to all addresses
+# If this is set to ::1 the webserver will listen to the IPv6 localhost address only
+listen_address = ::
```

Comment:
I would call this webserveraddress. So it is clear it belongs to webserverport.

Reply:
I changed it to `webserveraddress`.